evolver/calibration/actions.py

```diff
@@ -2,34 +2,44 @@
 from copy import deepcopy
 from typing import Any, Dict
 
+import pydantic
+
 
 class CalibrationAction(ABC):
+    class UserInput(pydantic.BaseModel): ...
+
     @abstractmethod
-    def execute(self, state: Dict[str, Any], payload: Dict[str, Any] = None) -> Dict[str, Any]:
+    def execute(self, state: Dict[str, Any], payload: UserInput) -> Dict[str, Any]:
         pass
 
 
 class DisplayInstructionAction(CalibrationAction):
+    class UserInput(pydantic.BaseModel):
+        confirmation: bool = pydantic.Field(True, description="Confirm that the instruction has been completed")
```

Comment:
small one, my fault - I think default should probably be `False`, forcing user to answer. Though now I wonder if this action even benefits from that? To get here they would have to press a button or something?

Reply:
Just a checkbox, so if you instruct the user to wait for environment to equilibriate for example, they have to click a checkbox confirming they did that step...

Reply:
and yeah, duh default of True makes no sense will fix.

Reply:
actually on reflection i think this is unecessary friction, the user can be trusted to follow instructions, and the confirmation checkbox  doesn't guarantee it anyway.

Reply:
yeah False is the correct default but really this isn't really useful for an instruction step. I was thinking it'd be good to force the user to confirm the instruction was followed but i think it's just friction and will be irritating.